### ISSUE TITLE
Changes could not be activated. 

### ISSUE THREAD

**Issue by AngelNev** [Stachelritter/CheckMK-PowerShell#12]
Ich nutze folgendes Skript mit dem psm1:

$CmkConnection  = Get-CMKConnection -Hostname $CmkSrvHostName -Sitename $CmkSiteName -Username $CmkUsername -Secret $CmkPassword 
$CmkHostExists  = Get-CMKHost -Connection $CmkConnection -HostName $HostName -ErrorAction SilentlyContinue

if ($CmkHostExists){
    Write-Host "$HostName already exists"
} else {
    Write-Host "Host does not exist, create $HostName"
    $CmkHostNew = New-CMKHost -Connection $CmkConnection -HostName $HostName -FolderPath '~test~windows~'

    Write-Host "Activate changes."
    $CmkPendingChanges = Get-CMKPendingChanges -Connection $CmkConnection

    Invoke-CMKChangeActivation -PendingChanges $CmkPendingChanges -ForceForeignChanges -Connection $CmkConnection

}

leider erhalte ich immer die fehlermeldung:
Get-CMKConnection: Cannot process argument transformation on parameter 'IfMatch'. Cannot convert value to type System.String.
Invoke-CMKChangeActivation: C:\Users\Userx\Documents\ms\PsCmkAgent\Checkmk-Tests.ps1:35:5
Line |                                                                                                                  
  35 |      Invoke-CMKChangeActivation -PendingChanges $CmkPendingChanges -Fo …                                         
     |      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                           
     | Changes could not be activated. Error message: Cannot bind argument to parameter 'Connection' because it is null.

**Comment by Stachelritter** [comment]
Ja, das geht bei mir (Powershell 5.1.17763.6414). Mit Powershell 7 habe ich das Modul nicht getestet. Welche Version verwendest du?
Ich vermute die Variable $CmkPendingChanges enthält kein Attribut "Etag" bei dir?
